src/_ToggleSynapseObjects.scss

```diff
@@ -0,0 +1,16 @@
+// Import the theme css
+@use 'variables' as Portal;
+
+.ToggleSynapseObjects {
+  margin-top: 30px;
+  .reactSwitchContainer{
+    width: 60px;
+    position: absolute;
+    right: 0px;
+    margin-top: -54px;
+  }
+  color: Portal.$primary-action-color;
```

Comment:
Use the themed primary action color (for the selected view), but override with a light grey when unselected.